projects/m4/003-concatenate-multiple-files/solutions/EvaLateltin/solutin-003-concatenate-multiple-files.js

```diff
@@ -0,0 +1,13 @@
+const fs = require("fs").promises;
+const inputFiles = ["file1.txt", "file2.txt", "file3.txt"]; // insert in the array the files that will be concatenated
+
+// The fist paramether is the array of the files that will be concatenated, the second paramether is the file where all the contented will be concatenated.
+function concatenateFiles(array, output) {
+  array.forEach((file) => {
+    fs.readFile(file, "utf8")
```

Comment:
Are you sure that the concatenation always happens in order?
<img width="1093" alt="image" src="https://user-images.githubusercontent.com/74595044/215261427-c56221b0-c0d6-4f30-a074-fb9d0b9bccd1.png">
Using promises you don't have this certainty. ;-)